src/client/components/Story/Body.js

```diff
@@ -37,6 +37,10 @@ export function getHtml(body, jsonMetadata = {}, returnType = 'Object', options
   const htmlReadyOptions = { mutate: true, resolveIframe: returnType === 'text' };
   parsedBody = remarkable.render(parsedBody);
   parsedBody = htmlReady(parsedBody, htmlReadyOptions).html;
+  parsedBody = parsedBody.replace(
+    /<a href="https:\/\/d.tube.#!\/v\/[^/"]+\/[^/"]+"><img src="[^"]+"\/><\/a>/g,
```

Comment:
Might be good to add this to regex helpers (`/src/client/helpers/regexHelpers.js`)